docs/UserGuide.md

```diff
@@ -238,4 +252,5 @@ Action     | Format, Examples
 **List**   | `list`
 **Help**   | `help`
 **Seed**   | `seed`
+**Sort**   | `sort parameter/order` <br> e.g., `sort n/ascending`
```

Comment:
Just a reminder to update this section with any changes you decide to make from the previous section